.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
Are we sure we want blank issues enabled? I find that if they're enabled, a lot of people in the community will just ignore any templates.

Reply:
I enabled those for the time being, so we can find out if our templates are not too restrictive.